File: SimActivationFrontEnd/src/app/models/simVerification.ts
export class SimVerificationModel {
    simNumber: string;
    serviceNumber: string;
    userID: number;
    isVerified: boolean;
    message: string;

    constructor(simNum: string, serviceNum: string, verified: boolean = false, msg: string = "Empty", userId: number = -1) {
        this.simNumber = simNum;
        this.serviceNumber = serviceNum;
        this.isVerified = verified;
        this.message = msg;
        this.userID = userId;
    }
}

File: SimActivationBackEnd/SimActivationMS/src/main/java/com/ab/sim/activation/dto/SimVerification.java
package com.ab.sim.activation.dto;

import javax.validation.constraints.NotNull;

public record SimVerification(
        @NotNull(message = "Sim Number is required")
        String simNumber,
        @NotNull(message = "Service Number is required")
        String serviceNumber,
        boolean isVerified,
        String message,
        Integer userID
) {
}
